nestkernel/event_delivery_manager.cpp

```diff
@@ -571,6 +571,27 @@ EventDeliveryManager::deliver_events_( const thread tid, const std::vector< Spik
 
         kernel().connection_manager.send( tid, syn_id, lcid, cm, se );
       }
+      else if ( spike_data.get_tid() == MAX_TID )  // handle compressed spikes
```

Comment:
Is it sensible to use `MAX_TID` to identify compressed spikes? How about a negative number? Are IDs signed?

Reply:
no, it's not. this should be fixed before merging into master, thanks @H27CK